src/api/kube.ts

```diff
@@ -1404,6 +1414,52 @@ export class KubeHelper {
     }
   }
 
+  async getOAuthClientAuthorizations(name: string): Promise<string[]> {
+    const customObjectsApi = KubeHelper.KUBE_CONFIG.makeApiClient(CustomObjectsApi)
+    try {
+      const { body } = await customObjectsApi.listClusterCustomObject('oauth.openshift.io', 'v1', 'oauthclientauthorizations')
+
+      if (!body.items) {
+        return []
+      }
+      const oauthClientAuthorizations = body.items as any[]
+      return oauthClientAuthorizations.filter(o => o.metadata.name && o.clientName === name)
```

Comment:
Why do we need to check if `o.metadata.name` exists ?

Reply:
I get the name of `oauthClientAuthorizations` from `o.metadata.name` to delete it.